okhttp-logging-interceptor/src/test/java/okhttp3/logging/HttpLoggingInterceptorTest.java

```diff
@@ -532,7 +532,7 @@ private void bodyGetNoBody(int code) throws IOException {
         .assertNoMoreLogs();
   }
 
-  @Test public void bodyResponseNotIdentityEncoded() throws IOException {
+  @Test public void bodyResponseGzipEncoded() throws IOException {
```

Comment:
Can you also check the normal response body is still valid in this test.  It's kind of the most critical thing for this test.

Reply:
Yep, good point - Done.